Cargo.toml

```diff
@@ -30,6 +30,7 @@ autocfg = "1"
 
 [target.'cfg(any(unix, target_os = "fuchsia", target_os = "vxworks"))'.dependencies]
 libc = "0.2.77"
```

Comment:
Can we move this to dev-deps?

Reply:
It's still used by the kqueue backend. I can make it kqueue only if you want.